velox/type/Variant.h

```diff
@@ -396,11 +396,12 @@ class variant {
     }
   }
 
-  std::string toJson(const TypePtr& type = nullptr) const;
+  std::string toJson(const TypePtr& type) const;
 
-  /// Used by python binding, do not change signature.
+  // Used by python binding, do not change signature.
```

Comment:
nit: keep `///` as this in a header class function.

Reply:
Fixed.